chroma-manager/tests/integration/core/api_testcase_with_test_reset.py

```diff
@@ -908,7 +910,11 @@ def create_zpools(self):
                                       server0['fqdn'],
                                       'export zfs device %s' % zfs_device)
 
-            self.execute_simultaneous_commands(['partprobe', 'udevadm settle'], fqdns, 'sync partitions')
+                # only partprobe the devices we are cleaning, as we can get
+                # EBUSY for the root disk for example
+                self.execute_simultaneous_commands(['partprobe %s' %
```

Comment:
is the indentation change deliberate?

Reply:
It is, but I can see that it shouldn't be as it's a parallel execution over `fqdns`.

But it was necessary in my Vagrant cluster testing, which is where I was seeing the EBUSY in the first place.  IOW, I was still getting failures when this was not indented.  Unfortunately I don't recall exactly what the error was.  Let me see if I can reproduce it.